SonarQube.Bootstrapper/BuildAgentUpdater.cs

```diff
@@ -52,6 +52,12 @@ public bool TryUpdate(string hostUrl, string targetDir, ILogger logger)
                         return false;
                     }
 
+                    if (e.Status == WebExceptionStatus.NameResolutionFailure)
+                    {
+                        logger.LogError(Resources.ERROR_UrlNameResolutionFailed, hostUrl);
+                        return false;
+                    }
+
```

Comment:
Fit and finish: improve the UX if the specified URL can't be found. Provides a more specific error message.